tests/scripts/check-names.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env python2
```

Comment:
The script is expected to work on Linux and OS X. `nm` isn't available on Windows, so it won't work there.

Reply:
Unless we want to tie this script to a specific Python version, we can just say ```#!/usr/bin/env python``` to allow working with whatever is default. That would need testing with both versions.

Reply:
@mazimkhan: I disagree because I think this script was written for python 2, but if the default is python 3 then it probably wont work because these two versions are not compatible.

Reply:
> That would need testing with both versions.

I did mention this.

Reply:
This was a mistake on my part, this should just be `python`. I was testing it with both versions and mistakenly left it as `python2`. The script is compatible with both python 2 & 3.